homeassistant/components/overkiz/light.py

```diff
@@ -79,8 +79,9 @@ def rgb_color(self) -> tuple[int, int, int] | None:
     @property
     def brightness(self) -> int | None:
         """Return the brightness of this light (0-255)."""
-        if brightness := self.executor.select_state(OverkizState.CORE_LIGHT_INTENSITY):
-            return round(cast(int, brightness) * 255 / 100)
+        value = self.executor.select_state(OverkizState.CORE_LIGHT_INTENSITY)
+        if value is not None:
```

Comment:
```suggestion
        if (brightness := self.executor.select_state(OverkizState.CORE_LIGHT_INTENSITY)) is not None:
```

Reply:
Yes, I do it on purpose, with black it will be formatted on 3 lines:

```python
        if (
            current_position := self.executor.select_state(OverkizState.CORE_DEPLOYMENT)
        ) is not None:
```

I’m not a big fan of this style.

Reply:
Just saying it can be done :D